packages/markers/src/browser/problem/problem-widget.tsx

```diff
@@ -170,6 +174,25 @@ export class ProblemWidget extends TreeWidget {
         </div>;
     }
 
+    /**
+     * Get the directory name of the node to be used as a description.
+     * - In a multiple root workspace, the root name is appended after the directory.
+     * @param node the marker info node.
+     *
+     * @returns the `description` to be used for display purposes.
+     */
+    protected getDescription(node: MarkerInfoNode): string {
```

Comment:
I wonder why it cannot be a part of `MarkerTreeLabelProvider.getLongName`?  It is the class responsible to compute such information.

Reply:
Good idea, I'll refactor and add a test for multiple root workspaces.